wg-manifests/charter.md

```diff
@@ -1,70 +1,50 @@
 # WG Manifests Charter
 
+
 This charter adheres to the conventions, roles and organization management
 outlined in [wg-governance].
 
 ## Scope
 
-- Provide a catalog (centralized repository) of Kubeflow application manifests.
-- Provide a catalog of third-party apps for common services.
+We simply (automatically) synchronize the application and dependencies manifests to then elaborately combine (configure)them for full platform experience.
+Providing a consistent and tested end-to-end multi-tenant experience is the most important task of the platform/manifests WG.
+To achieve this we maintain an extensive testing suite that covers most basic scenarios users would expect from a Platform for ML orchestration.
+We also provide the documentation regarding, but not limited to installation, extension, security and architecture to enable users to run their own ML Platform on Kubernetes.
+Users may choose to derive from platform/manifests to create so called distributions, which are opinionated to satisfy individual requirements.
+Users may also choose to install individual components without the benefits of the platform.
 
 ### In scope
 
 #### Code, Binaries and Services
 
-- Maintain tooling to automate copying manifests from upstream app repos.
-- Maintain a catalog that will allow users to install Kubeflow apps and
-  common services easily on Kubernetes, either on the cloud or on-prem, without
-  depending on external cloud services or closed source solutions. Those
-  manifests are deployed using `kubectl` and `kustomize` and include:
```

Comment:
We also need to ensure that `kustomize`/`kubectl` remains the primary "scope" of the manifests repo, as this is what all existing distributions/users are based on.

Also, are we allowing other deployment tools beyond `kubectl` and `kustomize` (e.g. `helm`, `argo cd`, `flux cd`), because this is a big scope change if so?